### DISCUSSION TITLE
rethink-type-url-dns/

### DISCUSSION THREAD

**Discussion by giscus** [n0vad3v/blog-comments#9] [Ideas]
# rethink-type-url-dns/

作为一个没有去过大厂的不怎么会写代码的萌新，我总是认为面试的过程是一个验证对方水平以及互相探讨问题并以此来确认对方学习能力的过程，而非一个应

https://nova.moe/rethink-type-url-dns/

**Comment by zh30** [comment]
很棒 👍

**Reply by Nova42x** [reply]
学习了❤️
